mod-users/src/main/resources/volaris/mod-users/features/profile-picture.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Profile-picture tests
+
+  Background:
+    * call read('classpath:common/util/random_numbers.feature')
+    * call read('classpath:common/util/random_string.feature')
+    * call read('classpath:common/util/uuid1.feature')
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def profileIdNotUploaded = 'c58d129e-347a-4931-9ffa-a27a3fffa7a8'
+    * def headersUserOctetStream = { 'Content-Type': 'application/octet-stream', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json'  }
+
+
+  Scenario: Upload user profile picture
+
+    # prepare tenant
+    Given path '/users/configurations/entry'
+    When method GET
+    Then status 200
+    * def id = response.id
+    * def encryptionKey = response.encryptionKey
+
+    Given path '/users/configurations/entry/' + id
+    And request
+    """
+         {
+           "id": "#(id)",
+           "configName": "PROFILE_PICTURE_CONFIG",
+           "enabled": true,
+           "enabledObjectStorage": false,
+           "encryptionKey": "#(encryptionKey)"
+         }
```

Comment:
Please write test cases when profile picture feature is not enabled.
When profile picture enabled for different kind of storage.
Please also write scenario to check the profile picture that is added is getting linked to users json, that scenario should be added to the existing users API. 
CC: @SinghAdes